app/ix/ginas/models/converters/HTMLStringConverter.java

```diff
@@ -88,27 +95,22 @@ public String toStd(String str){
      */
     @Override
     public String truncate(String str, int maxBytes){
-        byte[] b = (str + "   ").getBytes();
-        if(maxBytes >= b.length){
+        if(maxBytes >= str.length()){
```

Comment:
I don't think this will work.  The previous code converts to bytes because some encodings take more than 1 byte (up to 4) for some characters. `str.length` is the number of characters which may be 1/4 the number of bytes in the worst case.

Reply:
I'm using str.length() here because of str.charAt(i-1) in for loop.